apps/voting/contracts/Voting.sol

```diff
@@ -479,6 +750,7 @@ contract Voting is IForwarder, AragonApp {
 
     /**
     * @dev Internal function to check if a vote is still open for both support and objection
+    * @param vote_ The queried vote
     * @return True if less than voteTime has passed since the vote start
     */
     function _isVoteOpen(Vote storage vote_) internal view returns (bool) {
```

Comment:
`!vote.executed` is redundant check, since after introduction of second phase a vote couldn't be executed earlier than `voteTime` has been passing